lib/opensusebasetest.pm

```diff
@@ -605,7 +605,7 @@ sub wait_grub {
     elsif (match_has_tag('encrypted-disk-password-prompt')) {
         # unlock encrypted disk before grub
         workaround_type_encrypted_passphrase;
-        assert_screen "grub2", 15;
+        assert_screen "grub2", 60;
```

Comment:
I would even go higher as 90 seconds. the problem is that sometimes in aarch64 we can miss the grub screen and go straight to the login screen (at least for unencrypted systems)